src/travel-details-map-screen/TravelDetailsMapScreenComponent.tsx

```diff
@@ -216,7 +222,12 @@ export const TravelDetailsMapScreenComponent = ({
       <View style={controlStyles.backArrowContainer}>
         <BackArrow
           accessibilityLabel={t(MapTexts.exitButton.a11yLabel)}
-          onBack={onPressBack}
+          onBack={() => {
```

Comment:
What if the user navigates back through some form of navigation? Will it still show the in-app review prompt?

(e.g. in some android devices you have the on-screen navigation button, or you can do it through gesture)

Reply:
True, I added extra code to handle this, good catch.